lib/Graph/Graph.cpp

```diff
@@ -682,6 +682,35 @@ Node *Function::createChannelShuffle(llvm::StringRef name, NodeValue input,
   return createReshape(name.str() + ".reshape2", T, inDims);
 }
 
+Node *Function::createSqueeze(llvm::StringRef name, NodeValue input,
+                              llvm::ArrayRef<size_t> axes) {
+  auto originalSize = axes.size();
+  GLOW_ASSERT(originalSize > 0 && "Parameter `axes` must be provided.");
+
+  ShapeVector shapeAxes(axes.begin(), axes.end());
```

Comment:
Please add a comment that describes what's going on here. 
// Sort and unique the values in the array because ...